src/commands/generate/types.ts

```diff
@@ -176,10 +179,18 @@ export default class GenerateTypes extends Base {
             )
         }
 
-        this.features = await fetchAllCompletedOrArchivedFeatures(
-            this.authToken,
-            this.projectKey,
-        )
+        this.features = await Promise.all([
```

Comment:
nit:
instead of the .then() you can just get the completedFeatures and staleFeatures as a return value and then have the this.features be a spread of both like you had before